WordPress/Tests/XSS/EscapeOutputUnitTest.php

```diff
@@ -63,6 +63,7 @@ public function getErrorList() {
 			205 => 1,
 			206 => 1,
 			207 => 1,
+			212 => ( PHP_VERSION_ID < 50300 ) ? 1 : 0, // PHPCS on PHP 5.2 does not recognize T_NOWDOC.
```

Comment:
A straight compare with a constant is one of the fastest operations in PHP.

That's why I backported the `PHP_VERSION_ID` constant in PHPCS so it could be used here instead of having to make two function calls `version_compare( phpversion(), '5.3', '>' ) )`.

PHPCS min requirement is PHP 5.1 and `PHP_VERSION_ID` wasn't introduced until PHP 5.2.7 ;-)